addons/im_livechat/models/discuss_channel.py

```diff
@@ -61,7 +61,7 @@ def _channel_info(self):
             } if channel.country_id else False
             if channel.livechat_operator_id:
                 display_name = channel.livechat_operator_id.user_livechat_username or channel.livechat_operator_id.display_name
-                channel_infos_dict[channel.id]['operator_pid'] = (channel.livechat_operator_id.id, display_name.replace(',', ''))
+                channel_infos_dict[channel.id]['operator'] = {'name': display_name, **channel.livechat_operator_id.mail_partner_format(fields={'id': True, 'write_date': True})[channel.livechat_operator_id]}
```

Comment:
still have an issue here, "name", should not be returned when it contains something else than name